tests/store/storage/test_localzarrstore.py

```diff
@@ -200,12 +200,16 @@ def test_match_chunking(store):
 
     store.delete_all()
 
+    # Let's add some data and then try and add some data with different chunking
     chunks = {"time": 4}
-    data_a_chunked = data_a.chunk(chunks)
     store.add(version="v0", dataset=data_a)
+
+    data_a_chunked = data_a.chunk(chunks)
     chunking = store.match_chunking(version="v0", dataset=data_a_chunked)
 
-    assert not chunking
+    # As the data we originally put in wasn't chunked then we get the full size of the time coordinate
+    # which is 31 here
+    assert chunking == {'time': 31}
```

Comment:
Will this test still be valid when the default chunking is included? Or would this need to be updated?

Reply:
Yeah it'll still be valid as this is a layer lower than the default chunking. This is just making sure the correct chunking is used.